lumigator/python/mzai/backend/Dockerfile

```diff
@@ -1,6 +1,18 @@
-ARG BASE_IMAGE=python:3.11.9-bookworm
-FROM $BASE_IMAGE
+FROM python:3.11-slim-bookworm
```

Comment:
will this still be `3.11.9` - do we need to specify the minor release for stability?